### PR TITLE
feat: add release_created per path in manifest PR

### PR DESCRIPTION
**Pull request by phated**
We were just implementing a release process on a vscode extension in our monorepo, and that has a separate release flow than anything else, so we'd only want the publish phase to run if a specific path was released.

This adds a `release_created` flag on each published package in a manifest release.

### CODE REVIEW

**Comment by phated** [comment]
I assume this commit was made with some github noreply email instead of the one I've signed the CLA with?

File: index.js [hunk 1]
```diff
@@ -50,6 +50,7 @@ async function runManifest (command) {
       if (!release) {
         continue
       }
+      core.setOuput(`${path}--release_created`, true)
```
Comment (bcoe) [inline index.js:53]:
```suggestion
      core.setOutput(`${path}--release_created`, true)
```

Comment (phated) [reply index.js]: Thanks. I did it from the GitHub UI 🤦

**Review by bcoe — APPROVED** [review]

**Comment by bcoe** [comment]
@phated this looks ready to land to me, have you signed the CLA?

**Comment by phated** [comment]
@bcoe thanks for fixing these tests! I think something is broken with the bot because when I log in, it says my CLA is signed.

**Comment by phated** [comment]
@googlebot I signed it!

**Comment by phated** [comment]
@bcoe could the issue be that I use a dot in my email for all my git commits, but google has it down as 1 word (that's how it shows when I log in)?

**Pull request merged**